moveit_core/collision_detection_bullet/include/moveit/collision_detection_bullet/bullet_integration/bullet_utils.h

```diff
@@ -82,15 +82,15 @@ inline bool acmCheck(const std::string& body_1, const std::string& body_2,
     }
     else
     {
-      ROS_DEBUG_STREAM_NAMED("collision_detection.bullet", "No entry in ACM found, collision check between "
-                                                               << body_1 << " and " << body_2);
+      ROS_DEBUG_STREAM_NAMED("collision_detection.bullet",
```

Comment:
In general, I like that the second argument is fit into the next line so that there is a clear separation between logname and message.